docs/quickstart.md

```diff
@@ -0,0 +1,98 @@
+# Quickstart
+
+This is a quickstart guide for getting Cluster API Operator up and running on your Kubernetes cluster.
+
+For more detailed information, please refer to the full documentation.
+
+## Prerequisites
+
+- [Running Kubernetes cluster](https://cluster-api.sigs.k8s.io/user/quick-start#install-andor-configure-a-kubernetes-cluster).
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for interacting with the management cluster.
+- [Helm](https://helm.sh/docs/intro/install/) for installing operator on the cluster (optional).
+
+## Install and configure Cluster API Operator
+
+### Configuring credential for cloud providers
+
+Instead of using environment variables as clusterctl does, Cluster API Operator uses Kubernetes secrets to store credentials for cloud providers. Refer to [provider documentation](https://cluster-api.sigs.k8s.io/user/quick-start#initialization-for-common-providers) on which credentials are required.
+
+This example uses AWS provider, but the same approach can be used for other providers.
+
+```bash
+export CREDENTIALS_SECRET_NAME="credentials-secret"
+export CREDENTIALS_SECRET_NAMESPACE="default"
+
+kubectl create secret generic "${CREDENTIALS_SECRET_NAME}" --from-literal=AWS_B64ENCODED_CREDENTIALS="${AWS_B64ENCODED_CREDENTIALS}" --namespace "${CREDENTIALS_SECRET_NAMESPACE}"
```

Comment:
it might confuse new people where to find `AWS_B64ENCODED_CREDENTIALS` in the command

Reply:
There is a link and note above that we use AWS in this specific example, I wanted to avoid having a full description of what needs to be done for the AWS provider, wdyt?

Reply:
Fair enough, fine as is